pkg/sink/sink.go

```diff
@@ -56,6 +55,11 @@ import (
 	v1 "knative.dev/pkg/apis/duck/v1"
 )
 
+const (
+	// EventProcessed is sent for CloudEvent compliant providers
+	EventProcessed = "dev.tekton.event.triggers.processed.v1"
```

Comment:
Shall we document this

Reply:
We didn't document CloudEvent Response till now. Probably, need to document not only this but CloudEvent Response behaviour.

Reply:
Sure 
Can you create a issue so that it will be tracked

Reply:
Sure 
Can you create a issue so that it will be tracked

Reply:
Done here: https://github.com/tektoncd/triggers/issues/1601